backend/inspirehep/indexer/cli.py

```diff
@@ -24,6 +24,11 @@
 LOGGER = structlog.getLogger()
 
 
+@click.group()
+def index():
+    """*INACTIVE* Deprecated command from INVENIO. *INACTIVE*"""
```

Comment:
what about all the other index management commands, like `init`, `delete`, etc.?

Reply:
index is whole group all commands like init delete etc are inside it. They won't be available at all when overwriting this group.

Reply:
```bash
[pazembrz@laptop backend]$ poetry run inspirehep --help
Usage: inspirehep [OPTIONS] COMMAND [ARGS]...

  Command Line Interface for Invenio.

Options:
  --version  Show the flask version
  --help     Show this message and exit.

Commands:
  access     Account commands.
  alembic    Perform database migrations.
  citations  Command for citations
  db         Database commands.
  files      Command group to handle file operations.
  fixtures   Command related to records in inspire
  importer   Command to import records.
  index      *INACTIVE* Deprecated command from INVENIO.
  indexer    Command group Inspire indexing & remap...
  instance   Instance commands.
  jobs       Command for jobs
  mailing    Command to handle mailing.
  migrate    Commands to migrate records from legacy.
  orcid      Commands to run ORCID tasks.
  pid        PID-Store management commands.
  records    Records management.
  roles      Role commands.
  routes     Show the routes for the app.
  run        Run a development server.
  shell      Runs a shell in the app context.
  sitemap    Command group to handle sitemap operations.
  tokens     OAuth2 server token commands.
  users      User commands.
  webpack    Webpack commands.
```

```bash
[pazembrz@laptop backend]$ poetry run inspirehep index --help
Usage: inspirehep index [OPTIONS] COMMAND [ARGS]...

  *INACTIVE* Deprecated command from INVENIO. *INACTIVE*

Options:
  --help  Show this message and exit.
```

Reply:
I'd not deprecate the whole group unless we move all these commands to our ``indexer`` module.

Reply:
:+1: